src/pages/HttpBin/index.js

```diff
@@ -45,28 +45,27 @@ const HttpBin = () => {
   const [responseIsLoad, setResponseIsLoad] = useState(true);
 
   const createRequest = () => {
-    return new Promise((resolve, reject) => {
-      const baseURL = 'https://httpbin.org/';
-      setTimeout(() => {
-        try {
-          axios({
-            method: methodType,
-            baseURL: baseURL,
-            data: bodyParameters,
-            url: methodType.toLowerCase(),
-            params: JSON.parse(queryParameters),
-          })
-            .then(({ data }) => {
-              resolve(data);
-            })
-            .catch((error) => {
-              reject(error);
-            });
-        } catch (error) {
-          reject(error);
-        }
-      }, 1000);
-    });
+    const baseURL = 'https://httpbin.org/';
+    try {
+      axios({
+        method: methodType,
+        baseURL: baseURL,
+        data: bodyParameters,
+        url: methodType.toLowerCase(),
+        params: JSON.parse(queryParameters),
+      })
+        .then((response) => {
+          setResponseIsLoad(true);
+          setResponse(response);
+        })
+        .catch((error) => {
+          setResponseIsLoad(true);
+          setResponse(error.message);
+        });
+    } catch (error) {
+      setResponseIsLoad(true);
```

Comment:
why is it `true`?

Reply:
The same as https://github.com/iviaks/english-games/pull/7#discussion_r488421528